templates/tab-articles.php

```diff
@@ -8,35 +8,42 @@
  */
 
 ?>
-
-<div class="panel"></div>
-<ul id="articles-target">
-	<li><label><input type="radio" name="articles-target" value="eds" checked="checked">EDS</label></li>
-	<li><label><input type="radio" name="articles-target" value="vera">Vera</label></li>
-</ul>
-<p>Also try: Browse journals by title, View article databases</p>
+<p>Search journals and articles</p>
+<form action="https://widgets.ebscohost.com/prod/search/" id="edssearch" method="get" data-target="eds">
+	<div class="hidden">
+		<input name="direct" value="true" type="hidden">
+		<input name="authtype" value="ip,guest" type="hidden">
+		<input name="type" value="0" type="hidden">
+		<input name="groupid" value="main" type="hidden">
+		<input name="site" value="eds-live" type="hidden">
+		<input name="profile" value="eds" type="hidden">
+		<input name="custid" value="s8978330" type="hidden">
+		<input name="bquery" value="" type="hidden">
+		<input name="facet" value="AcademicJournals,Magazines" type="hidden">
+	</div>
+	<div class="row">
+		<input type="text" name="uquery" placeholder="Search journals and articles...">
+		<input type="submit" value="Search">
```

Comment:
I would rather have the submit semantically last and work on visual arrangement with css. I think you can drop the "row" div and move submit after the limit selector.